data/words/english_words_of_yiddish_origin.json

```diff
@@ -0,0 +1,109 @@
+{
+  "description": "a list of words that have entered the English language from the Yiddish language, many of them by way of American English.",
+  "source": "https://en.wikipedia.org/wiki/List_of_English_words_of_Yiddish_origin",
+  "nouns": [
+    "Bagel",
+    "Blintz",
+    "Bris",
+    "Boychik",
```

Comment:
spelling error!
```suggestion
    "Boychiky",
```

Reply:
I don't know Yiddish, but searching only shows it without -y, eg. https://en.wiktionary.org/wiki/boychik